src/js/containers/Buttons.js

```diff
@@ -29,7 +29,8 @@ const mapStateToProps = (state) => {
             image: softButtons[0].image ? softButtons[0].image.value : undefined,
             imageType: softButtons[0].image ? softButtons[0].image.imageType : undefined,
             id: softButtons[0].softButtonID,
-            isTemplate: softButtons[0].image ? softButtons[0].image.isTemplate : null
+            isTemplate: softButtons[0].image ? softButtons[0].image.isTemplate : null,
+            isHighlighted: softButtons[0].isHighlighted
```

Comment:
a similar change needs to be applied to the second controlbaritem added around line 150

Reply:
@iCollin Applied in 351e590